### PR TITLE
BETA: Register stavkv.is-a.dev

### PR DESCRIPTION
**Pull request by stavkv**
Added `stavkv.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/stavkv.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "stavkv",
+        "email": "singhvikal891@gmail.com"
+    },
+    "record": {
+        "CNAME": "stavkv.vercel.app"
```
Comment (Omxpro) [inline domains/stavkv.json:7]: vercel isnt supported

**Review by Omxpro — CHANGES_REQUESTED** [review]

File: domains/stavkv.json (hunk 1 above)
Comment (stavkv) [reply domains/stavkv.json:7]: Update

Comment (Omxpro) [reply domains/stavkv.json:7]: update the pr

**Review by stavkv — COMMENTED** [review]
Updated to github pages

**Review by Omxpro — APPROVED** [review]

**Comment by Omxpro** [comment]
looks good

**Review by iostpa — APPROVED** [review]

**Pull request merged**